docs/runtime/management.adoc

```diff
@@ -573,6 +573,120 @@ Installs an application from url on all nodes.
 }
 ----
 
+=== Uninstall an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Uninstalls an application with specific application key.
+
+.Url
+ POST app/uninstall
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Start an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Starts an application with specific application key.
+
+.Url
+ POST app/start
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Stop an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Stops an application with specific application key.
+
+.Url
+ POST app/stop
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Subscribe to Server-Sent Events
+
+image:xp-760.svg[XP 7.6.0,opts=inline] The client initiates the SSE connection by using the media type `text/event-stream` in the `Accept` header. Then it gets updates automatically without requesting the server.
+
+.Url
+ GET app/events
+
+Currently that API supports the following events:
+
+- `list` - returns the details of applications which already installed
+- `installed` - returns the details of just installed an application
+- `state` - returns the details of just started or stopped an application
+- `uninstalled` - returns the `key` of just removed an application
+
+Once a client was subscribed will be sent the first event `list`.
```

Comment:
Better English:
Once a client is subscribed it will be sent the first event `list`.

Reply:
Done